lib/bmp280.ex

```diff
@@ -120,45 +121,48 @@ defmodule BMP280 do
 
     {:ok, transport} = Transport.open(bus_name, bus_address)
 
-    state = %{
-      transport: transport,
-      calibration: nil,
-      sea_level_pa: Keyword.get(args, :sea_level_pa, @sea_level_pa),
-      sensor_type: nil
-    }
+    state =
+      %{
+        transport: transport,
+        calibration: nil,
+        sea_level_pa: Keyword.get(args, :sea_level_pa, @sea_level_pa),
+        sensor_type: nil,
+        last_measurement: nil
+      }
+      |> query_sensor()
```

Comment:
It's better to move calls that can block out of `init/1` so that they happen in the `GenServer`'s process rather than in the process that starts the `GenServer`. There's normally no race condition since the `handle_continue(:continue, state)` function is guaranteed to be the first function called. I'm not sure about `:sys.get_state`. It really seems like it should follow the same rules, but I'd have to look at the `:gen_server` source code to see for sure. It's probably safer to switch back to `GenServer.call(server, :measure)`